src/UserSettingsPage.cpp

```diff
@@ -655,9 +655,9 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         formLayout_->addRow(uiLabel_);
         formLayout_->addRow(new HorizontalLine{this});
 
-        boxWrap(tr("Mobile mode"),
+        boxWrap(tr("Touchscreen mode"),
```

Comment:
Not sure, I want to call it that, since it may toggle other features in the future as well, like make the timestamp appear below the message or similar.

Reply:
That makes sense, but the current toggle really is a *touchscreen* mode, as I reworded it in the tooltip.

I feel that having a toggle for this makes sense for non-mobile, touchscreen devices. (although even better would be to just ignore touch events for text selection when they are received, maybe it can be done with overloading a callback or disconnecting a signal?).

Conversely, such visual improvements could be useful regardless of the platform, wouldn't they?

Reply:
> Although even better would be to just ignore touch events for text selection when they are received

I tried that first, but most touch screens emulate a mouse and you can't easily distinguish it. It just broke copying text completely, so I made the simple, no surprise solution of just letting the user toggle it instead of breaking a lot of random stuff.

I guess we can rename it, but my intentions for the toggle were different :3